incubation_process.md

```diff
@@ -41,20 +43,24 @@ The barrier for entering the Sandbox stage is intended to be low, so that it's e
 
 ## Stage 2: Incubation
 
-The Incubation stage is for projects that have reached a level of maturity that indicates they are ready to be used by end users. Incubation projects meet the main criteria of the OpenRail Association which make sure that the project is governed openly. Open governance means that decisions are taken in a transparent way and there is a path to become committer and maintainer based on the merits of contributions.
+The Incubation stage is for projects that have reached a high level of quality and maturity that indicates they are ready to be used by end users. Incubation projects meet the main criteria of the OpenRail Association which make sure that the project is governed openly. Open governance means that decisions are taken in a transparent way and there is a path to become committer and maintainer based on the merits of contributions.
+
+The Incubation stage focuses on quality criteria the project can reach on its own and doesn't require a high degree of external adoption or contributions yet.
 
 The barrier for entering the Incubation stage is intended to be high, so there is a high level of quality for projects which have reached this stage.
 
 ### Criteria
```

Comment:
We need to check the phrasing of the criteria, so that they are written in a consistent way. Maybe it would be good to adopt a common format which includes the rationale for the criteria, e.g. "[criteria], so that [benefit]".

Reply:
I changed the language now so that they specify the criteria and their rationale in a consistent way.